ast/src/lib.rs

```diff
@@ -48,7 +48,7 @@ use from_pest::FromPest;
 use std::{fs, path::PathBuf};
 
 pub struct LeoAst<'ast> {
-    ast: files::File<'ast>,
+    pub ast: files::File<'ast>,
```

Comment:
Why?

Reply:
Yes, there is no needs for it. Previously, I used it for testing linter. I revert it